src/devtools/client/webconsole/components/Input/JSTerm.tsx

```diff
@@ -132,7 +159,14 @@ export default function JSTerm() {
     }
 
     if (
-      [Keys.ENTER, Keys.TAB, Keys.ESCAPE, Keys.ARROW_RIGHT, Keys.ARROW_LEFT].includes(e.key as Keys)
+      [
+        Keys.BACKSPACE,
+        Keys.ENTER,
+        Keys.TAB,
+        Keys.ESCAPE,
+        Keys.ARROW_RIGHT,
+        Keys.ARROW_LEFT,
+      ].includes(e.key as Keys)
```

Comment:
This is irrelevant functionally, but it's more correct to cast array to `string[]` (from narrower to wider type) than it is to cast `e.key` to `Keys` (wider to narrower type).